libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -313,7 +317,16 @@ - (void)authenticateWithCompletionBlock:(SFOAuthPluginAuthSuccessBlock)completio
         } else if (failureBlock != NULL) {
             failureBlock(authInfo, error);
         }
-    }];
+    };
+
+    if (![SFUserAccountManager sharedInstance].currentUser) {
```

Comment:
I don't see how this could happen (`SFHybridViewController` being present, but `currentUser` being yanked out from underneath), but we can leave that check in there for now until we refactor this class in `6.0`.